_component_design/bulk-selection.md

```diff
@@ -0,0 +1,208 @@
+
+### Composition
+
+Bulk selection contains the following elements:
+
+- **Parent checkbox (required)**: When the user selects the parent checkbox, all children checkboxes are selected. When the user deselects the parent checkbox, all children checkboxes are deselected.
+- **Select All (optional)**: When the user clicks **Select All**, all records on the current page are selected.
+- **Deselect All (optional)**: When the user click **Deselect All**, all records on the current page are deselected.
+- **Number selected (optional)**: The number selected shows the number of objects selected in the table. For example, **3 Selected** means that out of 10 records displayed in the table, three of those records are currently selected by the user. If you include number selected, then also include the **Select All** option.
+- **Selection bucket (optional)**: You can add a [selection bucket]({{site.baseurl}}/components/selection-bucket.html) to help the user manage table selections across multiple pages.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"xxx"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/tables/bulk-selection/placeholder-image.png" width="444"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Style
+
+Use these specifications when you construct bulk selection.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"xxx"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/tables/bulk-selection/placeholder-image.png" width="444"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Spacing
+
+Use these spacing guidelines when you construct bulk selection.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"xxx"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/tables/bulk-selection/placeholder-image.png" width="444"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Behaviors
+
+Ensure that the bulk selection user interactions support the following behaviors:
+
+- When the user selects multiple (but not all) checkboxes individually, the parent checkbox changes to the indeterminate state and the **Select All** control becomes active.
```

Comment:
strike "and the select all..." Then add, "If you include the number selected feedback, then also include a select all"

Reply:
Done.